src/App.scss

```diff
@@ -0,0 +1,298 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;300;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;400;700&display=swap');
+@import './reset.css';
+
+html,
+body {
+  box-sizing: border-box;
+  font-family: 'Montserrat', sans-serif;
+}
+
+body {
+  background-color: #fafafa;
+}
+
+.app {
+  &.line {
+    height: 2px;
+    margin: 40px 100px 29px 100px;
+    border: solid 1px #e8e8e8;
+  }
+
+  &.main {
+    width: 100%;
+    margin: 0 auto;
+  }
+}
+
+.navbar {
+  width: 100%;
+  height: 95px;
+  background-color: #fff;
+  margin: 0 auto;
+
+  &.content {
+    max-width: 1100px;
+    border-radius: 4px;
+    display: flex;
```

Comment:
- [ ] [REQUIRED] Please, make sure that basic CSS bad practices are not used ([article for support](https://speckyboy.com/good-bad-css-practices) ) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
- no repeating of the same rule

Reply:
KIndly implement these suggested changes by the previous code reviewer, you can make a class for `display: 'flex' `and apply it on all parts where `display: 'flex'` is used